prow/cmd/deck/static/prow/prow.ts

```diff
@@ -571,7 +572,17 @@ function redraw(fz: FuzzySearch): void {
         r.appendChild(cell.state(state));
         if ((agent === "kubernetes" && pod_name) || agent !== "kubernetes") {
             const logIcon = icon.create("description", "Build log");
-            logIcon.href = `log?job=${job}&id=${build_id}`;
+            if (pod_spec == null || pod_spec.containers.length <= 1) {
+                logIcon.href = `log?job=${job}&id=${build_id}`;
+            } else {
+                const buildIndex = url.indexOf('/build/');
```

Comment:
When does the URL contain `/build/`? I don't see any examples of that on https://prow.k8s.io/data.js.

Reply:
It doesn't seem like any of our urls for jobs have `/build/`, however I saw this logic for the [spyglass button](https://github.com/kubernetes/test-infra/blob/af45e5598c0395ad9b249e0031cbc1270b0e068e/prow/cmd/deck/static/prow/prow.ts#L618-L625) here and thought that it might be useful for some backwards compatibility reason.  If that's not the case, then yeah it should probably be removed since it doesn't seem to be relevant now.